DOCS-3787: Add warning about renaming namespace (#4306)
viamrobotics/docs: docs/manage/reference/organize.md

### Can I rename my organization after its creation?

> Yes, you can rename your organization id you are an organization owner.

Reviewer: ```suggestion
Yes, you can rename your organization if you are an organization owner.
```

{{< alert title="Caution" color="caution" >}}

> If your organization has added modules or packages to the Viam Registry, their namespace will be changed in the Registry automatically.

Reviewer: ```suggestion
If your organization owns modules or packages in the Viam Registry, the namespaces will be changed in the registry automatically.
```
[nit] Style guide seems to do registry this way when not coupled with "Viam"
